src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -36,6 +36,7 @@
 
 #include "ngraph/ngraph.hpp"
 #include <ngraph/opsets/opset1.hpp>
+#include <openvino/opsets/opset13.hpp>
```

Comment:
Do you really need the whole opset13? Please add just required operations.

Reply:
done